### ISSUE TITLE
Cover group does not allow to use the up button

### ISSUE THREAD

**Issue by TheZoker** [home-assistant/frontend#8436]
**Checklist**

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

**Describe the issue you are experiencing**

I have to single cover entities, which work great on the frontend (see sceenshots in additional information).
Now I created a group containing those two covers. When I use the group within the frontend, the "up" button is disabled (see screenshot)

**Describe the behavior you expected**

Since the position of both covers within this group is set to always 50, I expect the group to also allow the up action.

**Steps to reproduce the issue**

1. Have two cover entities
2. Set both of them to `position_template: "50"`
3. Create a group containing both entities
...

**What version of Home Assistant Core has the issue?**

core-2021.2.3

**What was the last working version of Home Assistant Core?**

_No response_

**In which browser are you experiencing the issue with?**

Microsoft Edge 89.0.774.27

**Which operating system are you using to run this browser?**

Windows 10 20H2

**State of relevant entities**

```yaml
cover.eg_links	
     state: open
     attributes:
          current_position: 50
          friendly_name: EG Jalousie Links
          supported_features: 11
          device_class: shutter
cover.eg_rechts
     state: open
     attributes:
          current_position: 50
          friendly_name: EG Jalousie Rechts
          supported_features: 11
          device_class: shutter
cover.jalousie_erdgeschoss
     state: open
     entity_id: cover.eg_links, cover.eg_rechts
     attributes:
          friendly_name: Jalousie Erdgeschoss
          supported_features: 11
          device_class: shutter
```

**Problem-relevant frontend configuration**

```yaml
type: button
show_name: true
tap_action:
  action: more-info
entity: cover.jalousie_erdgeschoss
name: EG
```

**Javascript errors shown in your browser console/inspector**

```txt
# Paste your logs here.

```

### Configuration:
```yaml
cover:
  - platform: template
    covers:
      eg_links:
        device_class: shutter
        position_template: "50"
        friendly_name: "EG Jalousie Links"
        open_cover:
          service: switch.turn_off
          data:
            entity_id: switch.jalousie_links
        close_cover:
          service: switch.turn_on
          data:
            entity_id: switch.jalousie_links
        stop_cover:
          service: switch.turn_on
          data:
            entity_id: switch.jalousie_links
  - platform: template
    covers:
      eg_rechts:
        device_class: shutter
        position_template: "50"
        friendly_name: "EG Jalousie Rechts"
        open_cover:
          service: switch.turn_off
          data:
            entity_id: switch.jalousie_rechts
        close_cover:
          service: switch.turn_on
          data:
            entity_id: switch.jalousie_rechts
        stop_cover:
          service: switch.turn_on
          data:
            entity_id: switch.jalousie_rechts
  - platform: group
    name: Jalousie Erdgeschoss
    entities:
      - cover.eg_links
      - cover.eg_rechts
```

### Screenshots:
![Screenshot_2021 02 22_23h27m05s_009_](https://user-images.githubusercontent.com/1368405/108838487-fc842500-75d3-11eb-9bfc-0ea7e18855fa.png)
![Screenshot_2021 02 22_23h27m13s_010_](https://user-images.githubusercontent.com/1368405/108838490-fdb55200-75d3-11eb-9823-e6f179a1a725.png)
![Screenshot_2021 02 22_23h27m26s_011_](https://user-images.githubusercontent.com/1368405/108838491-fdb55200-75d3-11eb-9526-2314bf92f5cc.png)

**Comment by spacegaier** [comment]
A cover group works fine for me when using regular covers (not templated ones).  Can you config Lovelace to render the position as the secondary info for your group to see what it has in there at runtime.

**Comment by TheZoker** [comment]
I created a test card with this config:
```yaml
type: entities
entities:
  - entity: cover.eg_rechts
    secondary_info: position
  - entity: cover.jalousie_erdgeschoss
    secondary_info: position
```

And this is the result:
![Screenshot_2021 02 25_12h56m06s_004_](https://user-images.githubusercontent.com/1368405/109150199-d5ab2780-7768-11eb-88bc-8d826ac49ca4.png)

So as seen in the attribute list, it seems that the group does not have a position at all. I think the best way to handle this issue would be, that if there is no position available, both controls are enabled

**Comment by spacegaier** [comment]
Two points / thoughts:
1. Why did you fix the `position_template` to 50? That would only make sense if the covers cannot actually be moved and will forever stay in that position.
2. My initial feeling is that this is more of a backend topic, because I think that the group cover integration should return the correct group position (as for example the light group does for brightness values), also for template covers. Your code proposal feels more like a band aid  for a backend gap/bug.

**Comment by TheZoker** [comment]
To the first point: 
So the issue is, that the covers are controlled by dumb switches. So they don't know if the cover is open or closed. 
Now Home Assistant tries to determine, if the cover is open or closed based on the last command (e.g. is the user opens the cover, HA thinks that the cover now is open. While this might good in theory, it does not, if the covers can also be controlled by an independent remote control. Because now HA does not know the state anymore and also can't assume it. 

So if I remove the `position_template` property from the config entry, the `down` button will be disabled if HA thinks, that the cover is already down. But if someone used the remote control to open the cover, HA does not know and now I can't close the cover, since HA thinks they are still down, while there are actually not. By setting the `position_template` to 50, HA does not try to assume the state and always has both control buttons enabled. Therefore I can open/close the cover at all times, no matter the actual state. I hope this makes sense. 

And to the second point:
I told Bram this scenario and he told me, that this might be a frontend issue (since the only difference between the config entry and the group is the missing `current_position`, which I now added via customization). That's why I opened the issue here. 

To sum it up:
If I don't have the `position_template`/`current_position` , I can't control the covers proper. There are also two community entries, which describe the same problem.

Here for the position_template:
https://community.home-assistant.io/t/cover-button-open-close-unavailable/184614

And here for the group:
https://community.home-assistant.io/t/cover-groups/61765

**Comment by TheZoker** [comment]
Issue still exists...

**Comment by spacegaier** [comment]
Re-reading this issue now:

@bramkragten Would you be fine with adding the proposed check from above? Sounds reasonable to me.

`if (stateObj.attributes.current_position === undefined)`
